usaspending_api/agency/v2/views/subcomponents.py

```diff
@@ -69,7 +71,7 @@ def format_results(self, file_a_response, file_b_response):
                 }
                 for x in combined_response
             ],
-            key=lambda x: x.get(self.pagination.sort_key),
+            key=lambda x: (x.get(self.pagination.sort_key) is not None, x.get(self.pagination.sort_key)),
```

Comment:
Would recommend using something like `sort_with_null_last()` here so that we can start to be more consistent. I know in some other places we have added Null values to the end of results.